scss/components/_icons.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file
+ * Contains class rules for material icons with different font sizes
+ *
+ * Usage
+ * To add a icon to the HTML document add a <i> tag with class `material-icons`
+ * e.g. <h2> <i class="material-icons">face</i> Title </h2>
+ * To know better refer - https://google.github.io/material-design-icons/#using-the-icons-in-html
+ * To see the list of all available icons - https://material.io/tools/icons/?style=baseline
+ */
+
+.material-icons {
+  @extend %material-icons;
+  
+  /* Rules for sizing the icon. Refer https://google.github.io/material-design-icons/#sizing*/
+  &.md-18 { 
```

Comment:
All these sizes are not necessary in our case, as our designs are not made with material design in mind. So we will be using custom font sizes according to the design. So please remove these.